### ISSUE TITLE
Bug, Ignoring domain-search option at 2.0.2 (3395) 

### ISSUE THREAD

**Issue by logichard** [passepartoutvpn/passepartout-apple#247]
Hello,  
After 2.0.2 (3395) on iOS and macOS, app ignoring pushed additional domain-search options. It's looks like, only domain is applied.

**Comment by keeshux** [comment]
Correct. I don't remember why I used to parse that option or where I possibly saw it because OpenVPN doesn't recognize it.

Instead, use `dhcp-option DOMAIN` from now on. Multiple entries will be applied in the same order they appear.

**Comment by keeshux** [comment]
https://community.openvpn.net/openvpn/ticket/1209

**Comment by keeshux** [comment]
https://community.openvpn.net/openvpn/ticket/1209#comment:21
